GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/DatabaseTests/Return/ReturnTests.IsLateSubmission.cs

```diff
@@ -110,5 +114,36 @@ public void Is_False_When_ModifiedDate_Is_Late_And_OutOfScope(SectorTypes sector
             }
         }
 
+        [TestCase(SectorTypes.Public, ScopeStatuses.InScope)]
+        [TestCase(SectorTypes.Public, ScopeStatuses.PresumedInScope)]
+        [TestCase(SectorTypes.Private, ScopeStatuses.InScope)]
+        [TestCase(SectorTypes.Private, ScopeStatuses.PresumedInScope)]
+        [TestCase(SectorTypes.Public, ScopeStatuses.OutOfScope)]
+        [TestCase(SectorTypes.Public, ScopeStatuses.PresumedOutOfScope)]
+        [TestCase(SectorTypes.Private, ScopeStatuses.OutOfScope)]
+        [TestCase(SectorTypes.Private, ScopeStatuses.PresumedOutOfScope)]
+        public void IsFalse_When_ModifiedDate_Is_Late_And_ExcludeFromLateFlagEnforcement_Year(SectorTypes sector, ScopeStatuses scopeStatus)
+        {
+            foreach (int testYear in Global.ReportingStartYearsToExcludeFromLateFlagEnforcement)
+            {
+                // Arrange 
+                DateTime snapshotDate = sector.GetAccountingStartDate(testYear);
+                DateTime modifiedDate = ReportingYearsHelper.GetDeadlineForAccountingDate(snapshotDate).AddDays(2);
+
+                Organisation testOrganisation = sector == SectorTypes.Private
+                    ? OrganisationHelper.GetPrivateOrganisation()
+                    : OrganisationHelper.GetPublicOrganisation();
+
+                OrganisationScope testScope = ScopeHelper.CreateScope(scopeStatus, snapshotDate);
+
+                Return testReturn = ReturnHelper.CreateLateReturn(testOrganisation, snapshotDate, modifiedDate, testScope);
+
+                // Act
+                bool actual = testReturn.IsLateSubmission;

```

Comment:
I assume this line is here because we're trying to split the test into three. Would it be better if we merge this with the following line, so:
Assert.AreEqual(false, testReturn.IsLateSubmission).
Just because I wasn't sure what we meant by "actual".

Reply:
Yes, that's right. When running a test we have the expected result and the actual result. This applies to all the tests and that's why this variable was named like that. I'd rather keep it as it is cause the test looks clearer this way.

Reply:
Great explanation! I agree 👍.